### PR TITLE
Newsletter Enhancement

### PR DESCRIPTION
**Pull request by anojht**
WIP, will need a hand on a few things 😃

### CODE REVIEW

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs [hunk 1]
```diff
@@ -489,17 +529,15 @@ private void CreateMovieHtmlContent(StringBuilder sb, MovieResponseDto info)
...
                         banner = banner.Replace("http", "https"); // Always use the Https banners
                     }
-                    AddImageInsideTable(sb, banner);
 
-                    sb.Append("<tr>");
-                    sb.Append(
-                        "<td align=\"center\" style=\"font-family: sans-serif; font-size: 14px; vertical-align: top;\" valign=\"top\">");
+                    //GET BACKGROUND URL HERE AND CALL 
+                    AddBackgroundInsideTable(sb, $"https://image.tmdb.org/t/p/w1280/");
+                    AddPosterInsideTable(sb, banner);
```
Comment (anojht) [inline src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: Need to fetch TV background before this call

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs [hunk 2]
```diff
@@ -581,26 +635,26 @@ private void CreateMovieHtmlContent(StringBuilder sb, MovieResponseDto info)
...
-                    sb.Append("<tr>");
-                    sb.Append(
-                        "<td align=\"center\" style=\"font-family: sans-serif; font-size: 14px; vertical-align: top;\" valign=\"top\">");
 
-                    Href(sb, $"https://www.imdb.com/title/{info.externals.imdb}/");
-                    Header(sb, 3, t.Title);
-                    EndTag(sb, "a");
+                    //GET BACKGROUND URL HERE AND CALL 
+                    AddBackgroundInsideTable(sb, $"https://image.tmdb.org/t/p/w1280/");
+                    AddPosterInsideTable(sb, banner);
```
Comment (anojht) [inline src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: Need to fetch TV background before this call

File: src/Ombi.Notifications.Templates/Templates/NewsletterTemplate.html [hunk 3]
```diff
@@ -127,40 +149,40 @@
...
 
                         <!-- START MAIN CONTENT AREA -->
                         <tr>
                             <td class="wrapper" style="font-family: sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;" valign="top">
                                 <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;" width="100%">
                                     <tr>
                                         <td align="center">
-                                            <img src="{@LOGO}" width="400px" text-align="center" />
+                                            <img src="{@LOGO}" width="400px" text-align="center"/>
                                         </td>
```
Comment (anojht) [inline src/Ombi.Notifications.Templates/Templates/NewsletterTemplate.html]: Currently points to imgur link with white bg. Need a logo with transparent bg.

Comment (tidusjar) [reply src/Ombi.Notifications.Templates/Templates/NewsletterTemplate.html]: Done

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs (hunk 2 above)
Comment (tidusjar) [reply src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: Done

**Pull request merged**